src/Compilers/CSharp/Portable/Compiler/TypeCompilationState.cs

```diff
@@ -7,6 +7,8 @@
 using Microsoft.CodeAnalysis.PooledObjects;
 using Roslyn.Utilities;
 
+#nullable enable
```

Comment:
💡 These should be placed above the `using` directives for consistency and so developers know where to look when checking for files that do not have annotations

Reply:
Thanks, will make sure to fix.